kobo/apps/kobo_auth/models.py

```diff
@@ -87,3 +90,37 @@ def sync_to_openrosa_db(self):
             unique_fields=['pk'],
         )
         update_autofield_sequence(User)
+
+
+class DataCollectorGroup(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL
+    )
+    uid = KpiUidField(uid_prefix='dcg', primary_key=True)
+    name = fields.CharField(max_length=200)
+
+    def __str__(self):
+        return '{} ({})'.format(self.name, self.uid)
+
+
+class DataCollector(models.Model):
```

Comment:
Make it inherit from AbstractTimestampModel. 
It always useful for debugging purpose (to have the creation, modification dates).